src/DataTable/types.ts

```diff
@@ -52,6 +53,7 @@ export type TableProps<T = RowRecord> = {
 	onChangePage?: ChangePage;
 	onChangeRowsPerPage?: ChangeRowsPerPage;
 	onRowClicked?: (row: T, e: React.MouseEvent) => void;
+	onRowHovered?: (row: T, e: React.MouseEvent) => void;
```

Comment:
Would calling this `onRowHover` be more appropriate since it's essentially `onMouseEnter`? In other words you are hovering, not that you have already hovered

Reply:
Just a thought, but, should we just pass all the mouse events? https://www.w3schools.com/jsref/obj_mouseevent.asp